test/WebJobs.Extensions.Tests/Extensions/DocumentDB/DocumentDBUtilityTests.cs

```diff
@@ -20,27 +17,27 @@ namespace Microsoft.Azure.WebJobs.Extensions.Tests.Extensions.DocumentDB
     public class DocumentDBUtilityTests
     {
         [Fact]
-        public async Task ExecuteAndIgnoreStatusCode_Ignores_SpecifiedStatusCode()
+        public async Task RetryAsync_Ignores_SpecifiedStatusCode()
         {
             // Arrange
             string testString = "Method called!";
             string s = null;
             var ex = DocumentDBTestUtility.CreateDocumentClientException(HttpStatusCode.Conflict);
 
             // Act
-            await DocumentDBUtility.ExecuteAndIgnoreStatusCodeAsync(HttpStatusCode.Conflict, () =>
+            await DocumentDBUtility.RetryAsync<object>(() =>
             {
                 s = testString;
                 throw ex;
-            });
+            }, 0, HttpStatusCode.Conflict, HttpStatusCode.NotFound);
 
             // Assert
             Assert.Equal(testString, s);
             // The fact that it doesn't throw proves that it was ignored
         }
 
         [Fact]
-        public async Task ExecuteAndIgnoreStatusCode_DoesNotIgnore_OtherStatusCodes()
+        public async Task RetryAsync_DoesNotIgnore_OtherStatusCodes()
```

Comment:
Good point. I've added a test for null.

---
In reply to: [60487790](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/83#discussion_r60487790) [](ancestors = 60487790)